lepton/qemu.go

```diff
@@ -397,44 +401,55 @@ func (q *qemu) addOption(flag, value string) {
 	q.flags = append(q.flags, option)
 }
 
-func (q *qemu) addAccel() {
+func (q *qemu) setAccel(rconfig *RunConfig) error {
+	if rconfig.Accel || rconfig.Bridged {
+		return q.addAccel()
```

Comment:
I don't know why acceleration is being silently enabled in a bridged configuration, but now that the accel flag is true by default, if the flag is false it means the user explicitly disabled acceleration in the config, so I think we should not enable it if a bridged config is there. So I would drop the `rconfig.Bridged` part in the `if` condition.

Reply:
agree, done